backends/platform/ios7/ios7_osys_misc.mm

```diff
@@ -81,9 +86,13 @@ static inline void execute_on_main_thread_async(void (^block)(void)) {
 	delete[] text;
 
 	return u32String;
+#else
+	return Common::U32String();
+#endif
```

Comment:
For consistency you probably also want to add the `#if TARGET_OS_IOS` to `OSystem_iOS7::hasFeature` so that it returns false for `kFeatureClipboardSupport` in the tvOS case.